droidlet/tools/crowdsourcing/droidlet_static_html_task/issue_bonus.py

```diff
@@ -17,49 +20,85 @@
 db = LocalMephistoDB()
 data_browser = DataBrowser(db=db)
 
-def main(run_id) -> list:
-    logging.info(f"Initializing bonus script for Mephisto job {run_id}")
+s3 = boto3.client('s3')
+
+
+def issue_bonuses(task_name: str) -> list:
+    logging.info(f"Initializing bonus script for Mephisto task_name: {task_name}")
+
+    logging.info(f"Downloading interaction bonus records from S3...")
+    with open("bonus_records.csv", "wb") as f:
+        s3.download_fileobj('droidlet-hitl', 'bonus_records.csv', f)
+
+    logging.info(f"Building list of already issued bonuses...")
+    issued_units = []
+    with open('bonus_records.csv', newline='') as csvfile:
+        reader = csv.reader(csvfile)
+        for row in reader:
+            issued_units.append(row[1])  # unit_id should be unique to each interaction HIT
 
     # Get completed units from the run_id
-    units = db.find_units(task_run_id=run_id)
+    logging.info(f"Retrieving units from Mephisto based on task_name...")
+    units = DataBrowser.get_units_for_task_name(task_name)
```

Comment:
Does this work? I feel like it should be `data_browser.get_units_for_task_name(task_name)` instead of `DataBrowser.get_units_for_task_name(task_name)`